esp-hal/src/dma/mod.rs

```diff
@@ -780,9 +815,12 @@ pub enum DmaPeripheral {
     Mem2Mem15 = 15,
 }
 
+/// An enum describing the owner of the buffer pointed to be a DMA descriptor.
```

Comment:
```suggestion
/// An enum describing the owner of the buffer pointed to by a DMA descriptor.
```

As a non-native speaker, this sentence is very hard to parse. Is "be" a typo here?

Regardless, could we phrase this in a simpler way if we avoided passive voice here?

It's also pretty obvious that this is an enum, do you think we need to explicitly call this out in the documentation, too?

Reply:
It's a typo :smile:, I'll reword it.

> do you think we need to explicitly call this out in the documentation, too?

Fair, I wasn't too sure how to start the sentence. "An enum ..." seemed easy enough.